docs/unified-test-documentation/dasharo-compatibility/327-embedded_controller_flashing.md

```diff
@@ -0,0 +1,46 @@
+# Dasharo Compatibility: Embedded controller flashing
+
+## Test cases
+
+### Common
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+### ECF001.001 Flash EC firmware externally using Arduino
+
+**Test description**
+
+This test aims to verify whether there is the possibility to flash the DUT
+EC firmware externally using Arduino.
+
+**Test configuration data**
+
+1. `FIRMWARE` = coreboot
+
+**Test setup**
+
+1. Proceed with the [Common](#common) section.
+1. Prepare the necessary hardware according to the
+    [Prerequisites](../../../variants/novacustom_ns5x/ec-recovery/#prerequisites)
+    section.
+1. Prepare the firmware according to the
+    [Preparation](../../../variants/novacustom_ns5x/ec-recovery/#preparation)
+    section.
+
+**Test steps**
+
+1. Go throw [flashing](../../../variants/novacustom_ns5x/ec-recovery/#flashing)
```

Comment:
```suggestion
1. Go through [flashing](../../../variants/novacustom_ns5x/ec-recovery/#flashing)
```

Reply:
@Pokisiekk maybe if we perform any testing operation based on the external documentation, we should describe it in a normalized manners, like in this case: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31P-flash-write-protection/